app/screens/apps_form/apps_form_field.tsx

```diff
@@ -210,7 +200,7 @@ export default class AppsFormField extends React.PureComponent<Props, State> {
                     dataSource={dataSource}
                     options={options}
                     optional={!field.is_required}
-                    onSelected={field.multiselect ? this.handleMultioptionAutocompleteSelect : this.handleAutocompleteSelect}
+                    onSelected={this.handleAutocompleteSelect}
```

Comment:
`this.handleMultioptionAutocompleteSelect` now one function.  `AutocompleteSelect` will handle if value is multiselect